pynicotine/gtkgui/widgets/textview.py

```diff
@@ -49,7 +47,6 @@ class TextView:
 
     except TypeError:
         # Broken cursor theme, but what can we do...
-        log.add_debug("Cannot load cursors from theme, falling back to default cursor")
```

Comment:
You suggested this log message originally.

Reply:
We have finished debugging that issue now, so it is no longer required (and so, the log import is not needed anymore for this is the only call to it).

Iirc our conclusion was that some cursor themes use different naming schemes, and indeed there is nothing we can do about that.